cognite_toolkit/cdf_tk/load.py

```diff
@@ -838,15 +843,18 @@ def delete(self, ids: Sequence[str], drop_data: bool) -> int:
 
     def load_resource(self, filepath: Path, dry_run: bool) -> ExtractionPipeline:
         resource = load_yaml_inject_variables(filepath, {})
+
         if resource.get("dataSetExternalId") is not None:
             ds_external_id = resource.pop("dataSetExternalId")
             resource["dataSetId"] = self.ToolGlobals.verify_dataset(ds_external_id) if not dry_run else -1
+
         return ExtractionPipeline.load(resource)
 
-    def create(self, items: Sequence[T_Resource], drop: bool, filepath: Path) -> T_ResourceList | None:
-        try:
-            return ExtractionPipelineList(self.client.extraction_pipelines.create(items))
+    def create(self, items: Sequence[ExtractionPipeline], drop: bool, filepath: Path) -> ExtractionPipelineList:
+        extractionPipelineList = None
 
+        try:
+            extractionPipelineList = ExtractionPipelineList(self.client.extraction_pipelines.create(items))
```

Comment:
```suggestion
            extraction_pipelines = self.client.extraction_pipelines.create(items)
```
Items is a sequence so create should return an extractionPipelineList

And if you don't mind, please avoid camel case in Python